cpuprofilenode.go

```diff
@@ -17,6 +17,12 @@ type CPUProfileNode struct {
 	// The number of the column where the function originates.
 	columnNumber int
 
+	// The number of samples recorded in the function
+	hitCount uint
```

Comment:
Would you be open to using the comment from the v8 header file for these?

i.e.
```
// The count of samples where the function was currently executing.
hitCount uint
```

```
// The number of the function's source lines that collect the samples.
hitLineCount uint
```